google-services-plugin/build.gradle

```diff
@@ -7,16 +7,17 @@ buildscript {
     }
 }
 
-project.ext.group = 'com.google.gms'
+group = 'com.google.gms'
+version = '4.2.0'
```

Comment:
Removed `ext` so that Gradle would automatically pick up group and version and use them for both main artifact and marker artifact.